### PR TITLE
Accommodate streamableMedia as ServiceFile in Component default struc…

### PR DESCRIPTION
**Pull request by coblej**
…ture; addresses DDR-809.

### CODE REVIEW

File: config/initializers/active_fedora_base.rb [hunk 1]
```diff
@@ -37,6 +37,14 @@ def has_admin_policy?
       governable? && admin_policy.present?
     end
 
+    def can_have_streamable_media?
```
Comment (dchandekstark) [inline config/initializers/active_fedora_base.rb]: This method is redundant - use or alias `can_be_streamable?`.

File: config/initializers/active_fedora_base.rb [hunk 2]
```diff
@@ -37,6 +37,14 @@ def has_admin_policy?
...
+      datastreams.include? Ddr::Datastreams::STREAMABLE_MEDIA
+    end
+
+    def has_streamable_media?
```
Comment (dchandekstark) [inline config/initializers/active_fedora_base.rb]: Redundant method -use or alias `streamable?`.

File: app/models/component.rb [hunk 3]
```diff
@@ -71,9 +71,16 @@ def add_intermediate_file(structure, filegrp, div)
     structure.add_fptr(parent: div, fileid: file['ID'])
   end
 
-  def add_service_file(structure, filegrp, div)
```
Comment (dchandekstark) [inline app/models/component.rb]: I feel as though these `add_*_file` private methods should be public methods of `Ddr::Models::Structure`.

**Comment by coblej** [comment]
@dchandekstark Still doing some refactoring work on this PR so don't merge yet.

**Comment by coblej** [comment]
@dchandekstark Refactoring done.  Ready for your re-review.  If you and Travis are happy with it, please approve/merge.

**Pull request merged**